src/App.tsx

```diff
@@ -1,73 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
+import { Autocomplete } from './components/Autocomplete';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const showPersonInfo =
+    selectedPerson !== null
+      ? `${selectedPerson.name} (${selectedPerson.born} - ${selectedPerson.died})`
+      : 'No selected person';
```

Comment:
Its not wrong, but you might not need to verify if its null, just use JS negative

```js
const personInfo = !selectedPerson ? 'No selected person' : `${selectedPerson.name} (${selectedPerson.born} - ${selectedPerson.died})`
```